docs/ref/databases.txt

```diff
@@ -114,11 +114,16 @@ below for information on how to set up your database correctly.
 PostgreSQL notes
 ================
 
-Django supports PostgreSQL 12 and higher. `psycopg2`_ 2.8.4 or higher is
-required, though the latest release is recommended.
+Django supports PostgreSQL 12 and higher. `psycopg`_ 3.1+ or `psycopg2`_ 2.8.4+
+is required, though the latest releases are recommended.
```

Comment:
In the 4.2 release not it references that 2.x will likely be deprecated. I wonder if it's making that reference here too? Folk may read release notes once, but this section many times? If it's been thought about, then I think it's worth communicating about as clearly as possible.

The recommendation would then be to use the latest version of the 3.x series?

Reply:
I pushed edits. What do you think?

Reply:
Looks great. 👍